libs/remix-ui/terminal/src/lib/components/Table.tsx

```diff
@@ -158,21 +158,6 @@ const showTable = (opts, showTableHash) => {
             </td>
           </tr>
         ) : null}
-        {opts.hash ? (
```

Comment:
this should be removed only if this isn't a call, you can use`opts.isCalll`

Reply:
Do we need this displayed on the UI? We already display `transaction hash` in terminal output. It is a duplicate.